.travis.yml

```diff
@@ -100,12 +100,12 @@ script:
   #
   - restore_configs
   - opt_enable AUTO_BED_LEVELING_UBL DEBUG_LEVELING_FEATURE G26_MESH_EDITING ENABLE_LEVELING_FADE_HEIGHT EEPROM_SETTINGS EEPROM_CHITCHAT G3D_PANEL
-  - opt_enable_adv CUSTOM_USER_MENUS I2C_POSITION_ENCODERS BABYSTEPPING NANODLP_Z_SYNC
+  - opt_enable_adv CUSTOM_USER_MENUS I2C_POSITION_ENCODERS BABYSTEPPING BABYSTEP_ZPROBE_OFFSET DOUBLECLICK_FOR_Z_BABYSTEPPING NANODLP_Z_SYNC
   - build_marlin
   #
   # And with a Sled Z Probe
   #
-  - opt_enable Z_PROBE_SLED 
```

Comment:
Gah - it's like every iteration of Sublime Text 3 update I have to reset the prefs to nuke trailing whitespace !